src/renderer/components/cluster-manager/lens-views.ts

```diff
@@ -20,10 +20,9 @@ export async function initView(clusterId: ClusterId) {
     return;
   }
   logger.info(`[LENS-VIEW]: init dashboard, clusterId=${clusterId}`)
-  const cluster = clusterStore.getById(clusterId);
   const parentElem = document.getElementById("lens-views");
   const iframe = document.createElement("iframe");
-  iframe.name = cluster.contextName;
+  iframe.name = clusterId;
```

Comment:
This is don't needed anymore I guess. Name was used previously only for dev-purposes.
But `clusterId` also visible in frame's URL.

Reply:
It seems that integration tests needs a proper name here.